recipes/openal-soft/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "openal-soft" %}
+{% set version = "1.20.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kcat/{{ name }}/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: c32d10473457a8b545aab50070fe84be2b5b041e1f2099012777ee6be0057c13
+
+build:
+  number: 0
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/cmake/OpenAL/OpenALConfig.cmake                             # [unix]
+    - if exist %LIBRARY_LIB%\\cmake\\OpenAL\\OpenALConfig.cmake (exit 0) else (exit 1)  # [win]
```

Can you add any tests for the commands that are? Generally people use `--help` or `--version` like
```suggestion
    - if exist %LIBRARY_LIB%\\cmake\\OpenAL\\OpenALConfig.cmake (exit 0) else (exit 1)  # [win]
    - altonegen --version
    - alrecord --help
    - openal-info
```